rust/emitter/src/emitter.rs

```diff
@@ -707,6 +713,19 @@ impl InstructionWriter {
         self.emit_op(Opcode::JumpTarget);
     }
 
+    pub fn bytecode_offset(&mut self) -> BytecodeOffset {
+        BytecodeOffset { offset: self.bytecode.len() }
+    }
+
+    pub fn patch_jump_target(&mut self, jumplist: Vec<BytecodeOffset>) {
+        let target_offset = self.bytecode.len();
```

Comment:
`self.bytecode_offset()` might be better for consistency.
(so, there's only one place that uses `bytecode.len()` to get the current offset)